src/get-started/wordcount-example.md

```diff
@@ -125,13 +125,13 @@ To view the full code in Python, see **[wordcount_minimal.py](https://github.com
 * Writing output (in this example: writing to a text file)
 * Running the Pipeline
 
-The following sections explain these concepts in detail along with excerpts of the relevant code from the Minimal WordCount pipeline.
+The following sections explain these concepts in detail, using the relevant code excerpts from the Minimal WordCount pipeline.
 
 ### Creating the Pipeline
 
-The first step in creating a Beam pipeline is to create a `PipelineOptions` object. This object lets us set various options for our pipeline, such as the pipeline runner that will execute our pipeline and any runner-specific configuration required by the chosen runner. In this example we set these options programmatically, but more often command-line arguments are used to set `PipelineOptions`. 
+The first step in creating a Beam pipeline is to create a `PipelineOptions` object. This object lets us set various options for our pipeline, such as the pipeline runner that will execute our pipeline and any runner-specific configuration required by the chosen runner. In this example we set these options programmatically, but more often, command-line arguments are used to set `PipelineOptions`.
```

Comment:
This is no longer technically true. It's a required step, but can be performed at any point before execution if the options are provided to the `run` method (at least in java)

Reply:
I changed this to be more specific to the example: "In this example, the code first creates a `PipelineOptions` object."  this is a good point from a bigger picture and this is something that may need to be addressed on on some other pages also.